### PR TITLE
Fyle mobile app autocomplete: Changes for recently used icon, add new item

### PR DESCRIPTION
**Pull request by shishirams222**
This PR contains changes of:
  1. Remove header and add recently used icon for recently used items:
    
Screenshot 2021-03-27 at 12 25 15 AM

  2. Show `Add "item_name"` on entering value which is not within the list when user can add new ones (vendor field):
  
Screenshot 2021-03-27 at 12 23 51 AM

### CODE REVIEW

**Review by abhishek1234321 — APPROVED** [review]

File: src/app/shared/components/fy-select-vendor/fy-select-modal/fy-select-vendor-modal.component.html [hunk 1]
```diff
@@ -52,5 +56,14 @@
         </ng-container>
       </div>
     </div>
+    <!-- All Vendors -->
+    <div class="" *ngIf="(filteredOptions$|async)?.length > 0" [ngClass]="{'selection-modal--all-merchants__margin-top': value}">
```
Comment (Julias0) [inline src/app/shared/components/fy-select-vendor/fy-select-modal/fy-select-vendor-modal.component.html:60]: `[ngClass]="{'selection-modal--all-merchants__margin-top': value}"`

Why is this required?

File: src/app/shared/components/fy-select-vendor/fy-select-modal/fy-select-vendor-modal.component.scss [hunk 2]
```diff
@@ -31,6 +31,11 @@ $list-element-color: #333;
     color: $sub-title;
   }
 
+  &--label {
+    text-align: justify;
+    word-break: break-word;
```
Comment (Julias0) [inline src/app/shared/components/fy-select-vendor/fy-select-modal/fy-select-vendor-modal.component.scss:36]: avoid using text align and word break.
Use whitespace nowrap or flexbox to acheive this

File: src/app/shared/components/fy-select-vendor/fy-select-modal/fy-select-vendor-modal.component.scss [hunk 3]
```diff
@@ -59,21 +64,44 @@ $list-element-color: #333;
...
+
+  &--add-item-icon {
+    min-height: 22px;
+    min-width: 22px;
+    margin-right: 10px;
+  }
+
+  &--new-item-name {
+    text-align: justify;
+    word-break: break-word;
```
Comment (Julias0) [inline src/app/shared/components/fy-select-vendor/fy-select-modal/fy-select-vendor-modal.component.scss:105]: same as above

File: src/app/shared/components/fy-select-vendor/fy-select-modal/fy-select-vendor-modal.component.html (hunk 1 above)
Comment (shishirams222) [reply src/app/shared/components/fy-select-vendor/fy-select-modal/fy-select-vendor-modal.component.html:60]: Did not add this, it was there from before
